### DISCUSSION TITLE
Affected cookies CSV represents which cookies will be affected due to 3rd party depreciation?

### DISCUSSION THREAD

**Discussion by dhineshbabu95** [GoogleChromeLabs/ps-analysis-tool#593] [Support Forum] [answered]
does the Affected cookies CSV represents which cookies will be affected due to 3rd party depreciation?

also how can I find which cookies will be affected due to 3rd party depreciation via the plugin?

**Comment by milindmore22** [comment, marked as answer]
Hello @dhineshbabu95 

Yes, that’s correct! The `affected-cookies.csv` contains the list of cookies that might be affected due to 3PCD.

If you are using PSAT extension in Chrome with 3PCD enabled, look for cookies with a yellow highlight. These cookies are most likely impacted by the third-party cookie depreciation.

You can always use the PSAT filter functionality in extension to and filter by “Third Party” cookies, please refer to [extension wiki](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/Cookies-Table#filtering)

**Reply by milindmore22** [reply]
@dhineshbabu95 Additionally, please note that The `affected-cookies.csv` will be renamed to `cookies-with-issues.csv` from release of version 0.7.0

**Reply by dhineshbabu95** [reply]
@milindmore22 I found that i get different number when I perform the audit through the CLI VS plugin for the same URL. If that's the same on your end why is that? Does the CLI have some sort of limitation when fetching the right number or is the plugin's number might be high cause of some other reason?

**Reply by dhineshbabu95** [reply]
@milindmore22 also once 3PCD is set in motion won't all 3P cookies be affected?

or some might be still active? if so why is that?

**Reply by milindmore22** [reply]
@dhineshbabu95 
> I found that i get different number when I perform the audit through the CLI VS plugin for the same URL. If that's the same on your end why is that? Does the CLI have some sort of limitation when fetching the right number or is the plugin's number might be high cause of some other reason?

Yes, absolutely! there can be slight differences between the PSAT extension and the command-line tool (CLI).

Several factors can affect how cookies appear, like the testing environment, analysis duration, and user actions on the page.

To learn more about discrepancies between PSAT CLI and extension, please check our [wiki](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/PSAT-Command-Line-Interface#discrepancy-between-cli-and-extension).

**Reply by milindmore22** [reply]
@dhineshbabu95 
> Also once 3PCD is set in motion won't all 3P cookies be affected? or some might be still active? if so why is that?

Although all third-party cookies are generally affected by privacy initiatives, there’s a concept called “Cookies Having Independent Partitioned State” (CHIPS) that creates an exception, For a better understanding of CHIPS, you can refer to the [documentation](https://developers.google.com/privacy-sandbox/3pcd/chips).

**Reply by amedina** [reply]
Some 3P cookies may be temporarily accepted via heuristic rules. For more information on this, please check [this article](https://developers.google.com/privacy-sandbox/3pcd/temporary-exceptions/heuristics-based-exceptions)
